apis/Google.Cloud.Datastore.V1/Google.Cloud.Datastore.V1.IntegrationTests/DatastoreDbTest.cs

```diff
@@ -120,6 +121,50 @@ public void RunQuery_NoResults()
             Assert.Equal(MoreResultsType.NoMoreResults, singleResponse.Batch.MoreResults);
         }
 
+        [Fact]
```

Comment:
These integration tests should be removed now - they'll fail before we get there anyway (because we're throwing an exception).